include/kan/vfs.h

```diff
@@ -1,17 +1,22 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
-/* Copyright (c), 2022, KanOS Contributors */
+/* Copyright (c), 2023, KanOS Contributors */
 #ifndef _INCLUDE_KAN_VFS_H__
 #define _INCLUDE_KAN_VFS_H__
 #include <stddef.h>
 #include <stdint.h>
 
 #define VFS_FILENAME_LENGTH 255
 
+typedef uint32_t vfs_group_id;
```

Comment:
Why typedef?

Why not just leave it as uint32_t and change it to uid_t/gid_t later (when there is uid_t/gid_t defined)

Reply:
I made it typedef for code clarity, so that it can be reused later. I can change it to uid_t and gid_t

Reply:
Why not define them right away, say in uapi/kan/types.h?